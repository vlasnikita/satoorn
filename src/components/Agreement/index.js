import React, { Component } from 'react';
import agreement from 'Static/agreement.pdf'

class Agreement extends Component {

    render() {
        return (
            <iframe
                src={agreement}
                frameBorder="0"
                style={{
                    height: '100vh',
                    width: '100vw'
                }}
            />
        );
    }
}

export default Agreement