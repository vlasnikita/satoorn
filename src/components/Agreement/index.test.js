import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react'
import Agreement from './index'

it('should render auth error page correctly', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<Agreement />)
    expect(wrapper).toMatchSnapshot()
})