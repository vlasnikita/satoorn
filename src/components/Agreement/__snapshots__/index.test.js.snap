// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render auth error page correctly 1`] = `
<iframe
  frameBorder="0"
  src={Object {}}
  style={
    Object {
      "height": "100vh",
      "width": "100vw",
    }
  }
/>
`;
